discover/deployment/articles-dxp/01-installing-liferay/07-installing-liferay-on-tcserver.markdown

```diff
@@ -1,9 +1,6 @@
 # Installing @product@ on tc Server [](id=installing-liferay-on-tc-server)
 
-The easiest way to get @product@ running on tc Server is to [download a bundle](https://web.liferay.com/group/customer/downloads/portal/product). If
-that's not an option, you can install @product@ onto tc Server. In addition to a
-supported version of [tc Server](https://network.pivotal.io/products/pivotal-tcserver) (version 3 or
-higher), make sure you have acquired the following items (from [Liferay's customer portal](https://web.liferay.com/group/customer/dxp/downloads/digital-enterprise))
+@product@ can be installed onto tc Server. In addition to a supported version of [tc Server](https://network.pivotal.io/products/pivotal-tcserver) (version 3.1.x), make sure you have acquired the following items (from [Liferay's customer portal](https://web.liferay.com/group/customer/dxp/downloads/digital-enterprise))
```

Comment:
We try never to put version numbers in the docs (docs become hard to maintain as version numbers change). Instead, provide a link to the support matrix.